src/models/index.js

```diff
@@ -30,6 +30,5 @@ Object.keys(db).forEach(modelName => {
 });
 
 db.sequelize = sequelize;
-db.Sequelize = Sequelize;
```

Comment:
This export of the library `Sequelize` doesn't seem to be used anywhere. Alternatively, in other places, we can access it through `require('sequelize')`. Anyone know if there was a specific reason we had this?

Reply:
yeah idk why this exists, I guess we don't need it but it's not a big deal if we leave it for now